.pipelines/templates/.builder-release-template.yaml

```diff
@@ -221,6 +221,7 @@ steps:
   - bash: |
         echo MODE=$(MODE) && \
         captured_sig_version="$(cat vhdbuilder/packer/settings.json | grep "captured_sig_version" | awk -F':' '{print $2}' | awk -F'"' '{print $2}')" && \
+        SIG_IMAGE_NAME="$(cat vhdbuilder/packer/settings.json | grep "sig_image_name" | awk -F':' '{print $2}' | awk -F'"' '{print $2}')" && \
```

Comment:
will trust your bash :p

Reply:
it works I think :D at least within abe2e testing

Reply:
i think I also copied this from somewhere else up above in the same template